packages/fxa-settings/src/models/hooks.ts

```diff
@@ -82,3 +82,25 @@ export function useFtlMsgResolver() {
   const { l10n } = useLocalization();
   return new FtlMsgResolver(l10n, config.l10n.strict);
 }
+
+/**
+ * Hook to run a function on an interval.
+ * @param callback - function to call
+ * @param delay - interval in Ms to run, null to stop poll
```

Comment:
Hhmm, what does passing `null` do?
The `useEffect()` below (L99-101) looks like passing `!delay && delay !== 0` just returns early and is a possible noop.